app/src/services/api.ts

```diff
@@ -10,7 +10,7 @@ import {RootState} from '../app/store';
 
 // Create base query
 const baseQuery = fetchBaseQuery({
-  baseUrl: '/api/',
+  baseUrl: 'http://localhost:4040/api/',
```

Comment:
for a separate PR but I think we should consider some kind of config setup so we don't have to change hosts in multiple places